src/components/ProfilePage.tsx

```diff
@@ -214,15 +252,59 @@ export const ProfilePage = (props: ProfilePageProps) => {
                                 }
                             </div>
                             <div className='containers'>
-                                <a href={"mailto:" + userBundle?.userProfile?.userName + "@synapse.org"}><img className='contact-icon' src={MailIcon}/>{userBundle?.userProfile?.userName}</a>
+                                <a href={"mailto:" + userBundle?.userProfile?.userName + "@synapse.org"}>
+                                    <img className='contact-icon' src={MailIcon}/>
+                                    {userBundle?.userProfile?.userName}
+                                </a>
+                            </div>
+                            <div className='containers'>
+                                {!editing ? 
+                                    userBundle?.userProfile?.url &&
+                                     <a href={userBundle?.userProfile?.url}>
+                                         <img className='contact-icon' src={LinkIcon}/>{userBundle?.userProfile?.url}
+                                    </a> 
+                                    :
+                                    EditField('Website', url, setUrl)
+                                }
                             </div>
-                            {userBundle?.userProfile?.url && <div className='containers'>
-                                {!editing ? <a href={userBundle?.userProfile?.url}><img className='contact-icon' src={LinkIcon}/>{userBundle?.userProfile?.url}</a> : EditField('Website', url, setUrl)}
-                            </div>}
                         </div>
                     </Col>
                 </Row>
             </Container>
+
+            <Modal
+                className= 'bootstrap-4-backport'
+                show={openCropModal}
+                onHide={()=>setOpenCropModal(false)}
+                animation={false}
+                centered
+            >
+                <Modal.Body style={{width:'400px',height:'400px', margin:'20px auto'}}>
+                    <div>
+                        <Cropper
+                            image={image}
+                            crop={crop}
+                            zoom={zoom}
+                            aspect={1}
+                            cropShape='round'
+                            onCropChange={setCrop}
+                            onZoomChange={setZoom}
+                            onCropComplete={onCropComplete}
+                        />
+                    </div>
+                </Modal.Body>
+                <Slider
+					min={1}
+					max={3}
+					step={0.1}
+					value={zoom}
+                    onChange={(e, zoom) => setZoom(zoom as number)}
+                />
+                <div id='crop-btn-container' className='modal-btn-container'>
```

Comment:
I don't think you need an ID, could probably just use a class name (why is modal-btn-container not enough on its own?)